### ISSUE TITLE
How to reload without cache?

### ISSUE THREAD

**Issue by frankdekker** [MicrosoftEdge/WebView2Feedback#1930]
Currently the CoreWebView2.Reload method reloads the currently active page preserving the cache state. Is there a way to programmatically reload the page without keeping the cache?

It also seems that CTRL+F5 keeps the cache.

**Comment by hunkydoryrepair** [comment]
yes, must use CTRL-SHIFT-R to clear without cache, but I can't figure out how to do it in code, either, other than use javascript.

**Comment by frankdekker** [comment]
Could you give an example how you would reload it via javascript? My current experimentation is to send CTRL+SHIFT+R to the WebView2 window handle, not quite successful so far.

**Comment by hunkydoryrepair** [comment]
Well, it used to work by doing this:

```
ctlWebBrowser.ExecuteScriptAsync("window.location.reload(true)");
```

I find that is no longer sufficient. It reloads the main document but not javascript or css files.  The `true` parameter was never official so chromium must have dropped support for it.

**Comment by torgeirf** [comment]
We have now had 2 occasions where this feature would be beneficial. First occasion was an unintentional permanent redirect response from the server, and the second case was a corrupt js file. Both of these issues required either:

1) Deleting the user data folder
or
2) Opening dev tools (guiding the user via phone) and make him disable cache and reload again

Please consider a method to "hard reload" the page programatically.
